skrifa/src/scale.rs

```diff
@@ -215,6 +213,196 @@ impl Context {
     }
 }
 
+/// Builder for configuring a glyph scaler.
```

Comment:
This description requires a fair bit of font knowledge to parse. Can we describe it in more layperson-friendly terms w/o losing anything?

First attempt:

```Builds [`Scaler`]'s that give access to glyph outlines via [Scaler::draw] to your [`Pen`]s```

Reply:
Point taken but the entire scale module effectively dead code at this point. I only kept it here due to our guarantee to keep old APIs during a transition period for Skia. This did remind me that the whole thing should probably be `doc(hidden)` so it doesn’t appear in public docs in the next release.